apigee_edge_ui.module

```diff
@@ -89,86 +105,97 @@ function template_preprocess_developer_app_list_for_developer(array &$variables)
+  $suggestions = [];
+  if (isset($variables['type'])) {
+    $suggestions[] = 'apigee_edge_ui_list__' . $variables['type'];
   }
+
+  return $suggestions;
 }
 
 /**
  * Implements hook_form_alter().
  */
 function apigee_edge_ui_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
-  $app_edit_form_ids = [
-    'developer_app_edit_for_developer_form',
-    'developer_app_edit_form',
-  ];
+  $form_object = $form_state->getFormObject();
 
-  $app_add_form_ids = [
-    'developer_app_add_for_developer_form',
-    'developer_app_add_form',
-  ];
+  if ($form_object instanceof EntityForm) {
+    /** @var \Drupal\Core\Entity\EntityForm $form_object */
+    $entity_type = $form_object->getEntity()->getEntityTypeId();
 
-  if (in_array($form_id, $app_add_form_ids)) {
-    if (isset($form['api_products'])) {
-      $form['api_products']['#attributes']['class'][] = 'developer-app-form__products';
-    }
-    $form['#attributes']['class'][] = 'developer-app-form--add';
-  }
-  elseif (in_array($form_id, $app_edit_form_ids)) {
-    $app_label = \Drupal::service('entity_type.manager')
-      ->getDefinition('developer_app')
-      ->getSingularLabel();
-    $form['actions']['submit']['#value'] = t('Save @app', ['@app' => $app_label]);
-    $form['actions']['delete']['#title'] = t('Delete @app', ['@app' => $app_label]);
-    $form['actions']['delete']['#weight'] = 1;
-    $form['#attributes']['class'][] = 'developer-app-form--edit';
-
-    $detail_fields = [
-      'callbackUrl',
-      'description',
-      'displayName',
-    ];
-
-    // Add details wrapper to non-credential info fields.
-    $form['details'] = [
-      '#type' => 'fieldset',
-      '#title' => t('Details'),
-      '#tag' => 'div',
-      '#attributes' => [
-        'class' => [
-          'developer-app-form__details',
-        ],
-      ],
-    ];
+    if (in_array($entity_type, ['team', 'team_app', 'developer_app'])) {
+      $form['#attributes']['class'][] = 'apigee-edge-entity-form';
+      $label = \Drupal::service('entity_type.manager')
+        ->getDefinition($entity_type)
+        ->getSingularLabel();
 
-    foreach ($detail_fields as $field) {
-      $form['details'][$field] = $form[$field];
-      hide($form[$field]);
-    }
+      if (isset($form['api_products'])) {
+        $form['api_products']['#attributes']['class'][] = 'apigee-edge-entity-form__products';
+      }
 
-    if (isset($form['credential'])) {
-      foreach (Element::children($form['credential']) as $credential) {
-        $form['credential'][$credential]['#attributes']['class'][] = 'developer-app-form__credential';
-        $form['credential'][$credential]['api_products']['#attributes']['class'][] = 'developer-app-form__products';
+      if (isset($form['actions']['submit']) && empty($form['actions']['submit']['#value']->getArguments())) {
+        $action = $form['actions']['submit']['#value']->getUntranslatedString();
```

Comment:
I do not prefer retrieving the untranslated string from a translatable markup and using it like this. (if you check to core that method is rarely used.) What about

`t('@action @entity', ['@action' => $form['actions']['submit']['#value'], '@entity' => $label])` Translatable markup automatically gets casted to string.